types.go

```diff
@@ -1114,7 +1114,7 @@ func makeGoLangScanType(ti typeInfo) reflect.Type {
 	case typeVariant:
 		return reflect.TypeOf(nil)
 	default:
-		panic(fmt.Sprintf("not implemented makeGoLangScanType for type %d", ti.TypeId))
+		return reflect.TypeOf(new(interface{})).Elem()
```

Comment:
Move this common code into a helper function that also logs the type name so developers can see when this is happening if they hook up driver logs in their app.